File: Assignment_2/part1/section1/sum_error.js
// JavaScript: Calculate the sum of an array (deliberate syntax error)
function calculateSum(arr) {
    let total = 0;
    for (let num of arr) {
        total += num;
    // <-- missing closing brace for the for-loop
    return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = calculateSum(numbers);
console.log("Sum in JavaScript:", result);

File: Assignment_2/part1/section1/sum_error.py
# Python: Calculate the sum of an array (deliberate syntax error)
def calculate_sum(arr)               # <-- missing colon at end of def header
    total = 0
    for num in arr:
        total += num
    return total

numbers = [1, 2, 3, 4, 5]
result = calculate_sum(numbers)
print("Sum in Python:", result)
